dashboard.el

```diff
@@ -128,20 +133,28 @@
           (setq dashboard--section-starts nil)
           (mapc (lambda (els)
                   (let* ((el (or (car-safe els) els))
-			 (list-size
+												 (list-size
                           (or (cdr-safe els)
                               dashboard-items-default-length))
-			 (item-generator
+												 (item-generator
                           (cdr-safe (assoc el dashboard-item-generators))))
                     (add-to-list 'dashboard--section-starts (point))
-                    (funcall item-generator list-size)
+                    (setq max-line-length
+													(max max-line-length (or (funcall item-generator list-size) 0)))
                     (dashboard-insert-page-break)))
-		dashboard-items))
-	(dashboard-mode)
-	(goto-char (point-min))))
+								dashboard-items)
+					(if dashboard-center-content
```

Comment:
Please use `(when` for this instead if `(if (progn)`

Reply:
Fixed!